src/vs/workbench/contrib/debug/common/debugSchemas.ts

```diff
@@ -215,6 +215,11 @@ export const launchSchema: IJSONSchema = {
 						},
 						description: nls.localize('app.launch.json.compounds.configurations', "Names of configurations that will be started as part of this compound.")
 					},
+					stopAll: {
+						type: 'boolean',
+						default: false,
+						description: nls.localize('app.launch.json.compound.stopAll', "Whether to stop all of the compound sessions when any session is terminated.")
```

Comment:
I suggest to use the word "controls" and "manual". Thus
"Controls if manually terminating one session will stop all of the compound sessions"
Though your native english is better, so just take this as a suggestion.